app/linux_stuff/io.github.marco_calautti.DeltaPatcher.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 8.0-->
+  <id>io.github.marco_calautti.DeltaPatcher</id>
+  <name>Delta Patcher</name>
+  <summary>Applies and creates patches in xdelta format</summary>
+  <developer_name>Marco Calautti</developer_name>
+  <launchable type="desktop-id">io.github.marco_calautti.DeltaPatcher.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <description>
```

Comment:
I would keep only the first paragraph in the description, describing what the tool does. The other two are unnecessary and not relevant.